mk/aosp_optee.mk

```diff
@@ -50,7 +50,10 @@ BUILD_OPTEE_OS:
 		CFG_ARM64_core=$(OPTEE_CFG_ARM64_CORE) \
 		PLATFORM=$(OPTEE_PLATFORM) \
 		PLATFORM_FLAVOR=$(OPTEE_PLATFORM_FLAVOR) \
-		$(CROSS_COMPILE_LINE)
+		$(CROSS_COMPILE_LINE) \
+		CFG_TEE_CORE_LOG_LEVEL=3 \
```

Comment:
Yes, but people who use these builds usually want to develop their own TAs, so the default setting should be friendly to them, i.e. debuggable and logs viewable. Otherwise they'll just end up having to rebuild again. E.g. #1438 above.